### PR TITLE
New session handling

### PR DESCRIPTION
**Pull request by lawinslow**
@jread-usgs @aappling-usgs You guys want to check off this workflow before I do anymore?

### CODE REVIEW

File: R/get_watershed_WFS.R [hunk 1]
```diff
@@ -17,21 +17,24 @@
...
 }
 
-get_watershed_item = function(site, session){
+get_watershed_item = function(site, ...){
+	
+	session_check_reauth(...)
+	
   # session not needed since items are public
-  identifier <- query_item_identifier(scheme = get_scheme(), type = 'watershed', key = site,  session = session)
+  identifier <- query_item_identifier(scheme = get_scheme(), type = 'watershed', key = site)
```
Comment (aappling-usgs) [inline R/get_watershed_WFS.R]: I see session_check_reauth followed by note that auth isn't even needed...

File: R/get_watershed_WFS.R [hunk 2]
```diff
@@ -2,8 +2,8 @@
 #'@description get a WFS url from a site that has a 'watershed'
 #'
 #'@param site a site identifier
-#'@param session arguments passed to sbtools \code{query_item_identifier} for authentication. 
-#'Not needed if parent of sites is public 
+#'@param ... Additional parameters supplied to \code{\link[sbtools]{session_check_reauth}}
```
Comment (aappling-usgs) [inline R/get_watershed_WFS.R]: this seems fine. nice to have session out of all those arg lists.

File: R/get_watershed_WFS.R (hunk 1 above)
Comment (lawinslow) [inline R/get_watershed_WFS.R]: Talking with Jordan. We should assume auth is needed. That comment should be changed (and was contradictory before as we were passing in a session).

**Comment by lawinslow** [comment]
Ah, there we go. @aappling-usgs merge if you're cool with the changes. (also, Powstreams won't build correctly until this is merged).

**Comment by aappling-usgs** [comment]
huzzah! thanks, @lawinslow

**Pull request merged**